proto/axelar/evm/v1beta1/query.proto

```diff
@@ -156,3 +156,29 @@ message BytecodeRequest {
 }
 
 message BytecodeResponse { string bytecode = 1; }
+
+enum TokenType {
+  option (gogoproto.goproto_enum_prefix) = false;
+  option (gogoproto.goproto_enum_stringer) = true;
+
+  TOKEN_TYPE_UNSPECIFIED = 0
+      [ (gogoproto.enumvalue_customname) = "Unspecified" ];
+  TOKEN_TYPE_INTERNAL = 1 [ (gogoproto.enumvalue_customname) = "Internal" ];
+  TOKEN_TYPE_EXTERNAL = 2 [ (gogoproto.enumvalue_customname) = "External" ];
+}
+
+message ERC20TokensRequest {
+  string chain = 1;
+  TokenType type = 2;
+}
+
+message ERC20TokensResponse { repeated string assets = 1; }
+
+message TokenDetailsRequest {
+  string chain = 1;
```

Comment:
Should use `github.com/axelarnetwork/axelar-core/x/nexus/exported.ChainName`.

Reply:
This breaks the REST API endpoint, so keeping it as is